weekly-updates/Georgia-Aitkenhead/weekly-meetings-2020.md

```diff
@@ -2,20 +2,21 @@
 
 Georgia's weekly meeting notes.
```

Comment:
Looks like January meetings are still in here - please delete from this file. You've already moved them to 2021 so no additional action there 😄  (Note typos in dates for January meetings (2020 instead of 2021), but you're deleting them anyway!)